workshop/web-vitals-lib/demo-cdn/package.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "cwv-cdn",
+    "version": "1.0.0",
+    "description": "Measure CWV with library from CDN",
+    "main": "index.js",
+    "scripts": {
+      "start": "node index.js",
```

Comment:
In the case you just serving static files, please use http-server instead.
Like `http-server -o files/index.html`